library/entity/tracked_data/src/main/java/org/quiltmc/qsl/tracked_data/mixin/TrackedDataHandlerRegistryMixin.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.tracked_data.mixin;
+
+import com.mojang.logging.LogUtils;
+import net.minecraft.entity.data.TrackedDataHandler;
+import net.minecraft.entity.data.TrackedDataHandlerRegistry;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import org.jetbrains.annotations.Nullable;
+import org.quiltmc.loader.api.QuiltLoader;
+import org.quiltmc.qsl.base.api.util.TriState;
+import org.quiltmc.qsl.tracked_data.impl.QuiltTrackedDataInitializer;
+import org.slf4j.Logger;
+import org.spongepowered.asm.mixin.*;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(TrackedDataHandlerRegistry.class)
+public class TrackedDataHandlerRegistryMixin {
+	@Unique
+	private static final Logger quilt$LOGGER = LogUtils.getLogger();
+	@Unique
+	private static final boolean quilt$PRINT_WARNING = TriState.fromProperty("quilt.unknown_tracked_data_handler_warning").toBooleanOrElse(QuiltLoader.isDevelopmentEnvironment());
```

Comment:
```suggestion
	private static final boolean quilt$PRINT_WARNING = TriState.fromProperty("quilt.debug.unknown_tracked_data_handler").toBooleanOrElse(QuiltLoader.isDevelopmentEnvironment());
```

We probably should consider some kind of internal API for those so we can easily list them whenever needed.